src/test/java/converter/StringToUserConverterTest.java

```diff
@@ -0,0 +1,71 @@
+package converter;
+
+import model.User;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class StringToUserConverterTest {
+    private final StringToUserConverter stringToUserConverter = new StringToUserConverter();
+
+    @Test(expected = IllegalArgumentException.class)
+    public void testConvertEmptyString() {
+        stringToUserConverter.convert("");
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void testConvertBlankString() {
+        String actual = " ";
+
+        stringToUserConverter.convert(actual);
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void testConvertNullString() {
+        stringToUserConverter.convert(null);
+    }
+
+    @Test
+    public void testConvertUser() {
+        User actual = null;
+        User expected = null;
+
+        assertEquals("can be User: null", expected, actual);
+    }
+
+    @Test
+    public void testConvertWithDash() {
+        User actual = stringToUserConverter.convert("vany-kaka=dimon-maradona=Van-Clock-Dimon=rbIba-gold");
+
+        assertNotNull(actual);
+        assertEquals("can be name:Van-Clock-Dimon", "Van-Clock-Dimon", actual.getName());
```

Comment:
make variables for expected results

Reply:
User expected = ...
User actual = ...

assertEquasl(expected.getSmth(), actual.getSmth())